src/test/java/com/nerodesk/om/aws/AwsDocTest.java

```diff
@@ -49,4 +62,67 @@ public void conformsToEqualsHashCodeContract() {
             .suppress(Warning.TRANSIENT_FIELDS)
             .verify();
     }
+
+    /**
+     * AwsDoc can report its existence.
+     * @throws IOException If unsuccessful.
+     */
+    @Test
+    public void reportsExistence() throws IOException {
+        final Bucket bucket = Mockito.mock(Bucket.class);
+        final Ocket ocket = Mockito.mock(Ocket.class);
+        Mockito.doReturn(true).when(ocket).exists();
+        final String label = "exists";
+        Mockito.doReturn(ocket).when(bucket).ocket(label);
+        MatcherAssert.assertThat(
+            new AwsDoc(bucket, label).exists(),
+            Matchers.equalTo(ocket.exists())
```

Comment:
@alevohin not that important. the value of true a few lines above is irrelevant if I compare the way I do now :)